acc_test/resource_cluster_test.go

```diff
@@ -21,6 +21,8 @@ func AccClusterResourceTestSteps(t *testing.T, testAccProviders map[string]terra
 
 	oriConfig := fmt.Sprintf(string(validConfig), username, apiKey, hostname)
 	updatedConfig := strings.Replace(oriConfig, "testcluster", "newcluster", 1)
+	oldToNewVersionConfig := strings.Replace(updatedConfig, `version = "apache-cassandra-3.11.8.ic2"`, `version = "3.11.8"`, 1)
```

Comment:
Hmm, does all our existing test configs use the old format? If so, do you think they should be updated as well?

Reply:
Pretty much, that's something I can go through and change too.